dialogue-apache-hc5-client/src/main/java/com/palantir/dialogue/hc5/ApacheHttpClientChannels.java

```diff
@@ -403,7 +403,6 @@ public CloseableClient build() {
                     .setPoolConcurrencyPolicy(PoolConcurrencyPolicy.LAX)
                     // Allow unnecessary connections to time out reducing system load.
                     .setConnPoolPolicy(PoolReusePolicy.LIFO)
-                    .setConnectionTimeToLive(IDLE_CONNECTION_TIMEOUT)
```

Comment:
This one is the :hot_pepper:, it's a maximum total timeout for each connection. After a connections lifespan has exceeded 50 seconds it will be closed and never reused.